Update manual_annotation notebook (#35)
napari/napari-workshop-template: napari-workshops/notebooks/manual_annotation.md

- drawing 2D polygons to identify particular regions of interest with the **Shapes** layer
- painting labels to provide a pixel-wise annotation of an image with the **Labels** layer

> This tutorial will explore these three manual annotations in napari, using the same data from the previous image visualization tutorial. Once again, we will try to highlight the bidirectional communication between the viewer and the Python kernel.

Reviewer: ```suggestion
This tutorial will explore these three manual annotations in napari, using the same data from the [Bioimage visualization in Python tutorial](intro_bioimage_visualization.md). Once again, we will try to highlight the bidirectional communication between the viewer and the Python kernel.
```

I think this is the right link? People may not be reading these notebooks in order.

viewer = napari.Viewer()
```

> In this notebook we will use the same `cells3d` data set:

Reviewer: ```suggestion
In this notebook we will use the `cells3d` data set:
```

Author: it's the same data, just loaded a different way, so maybe we leave `same`?

Author: I kinda wonder if we should just switch everything to use the skimage data so people dont need the repo cloned with the data file. I think showing how to open a data file is important though for intro.

Reviewer: Maybe open an issue with that suggestion, and we can follow up on it

```{code-cell} ipython3
from skimage.data import cells3d

image_data = cells3d()  # shape (60, 2, 256, 256)

membranes = image_data[:, 0, :, :]
nuclei = image_data[:, 1, :, :]

viewer.add_image(nuclei)
```

This loads the 3D data into the napari viewer on the 30th slice. Let's take a screenshot
for record keeping.

```{code-cell} ipython3
nbscreenshot(viewer)
```

## Annotating dividing and non-dividing cells using the points layer

> One simple task that a biologist or bioimage analyst might be interested in annotating each cell as diving or non-dividing.

Reviewer: ```suggestion
One simple task that a biologist or bioimage analyst might be interested in is annotating each cell as dividing or non-dividing.
```

Author: But *diving* cells are so interesting!! 🤣

Reviewer: I had a genuine chuckle out of the diving cells, with their swimming caps and everything

In order to do this we are going to add two points layers to the viewer, one called `dividing` and one called `non-dividing` and set some basic properties on these layers.

> You can add the layers using the new points button in the middle of the left panel of the viewer (left-most button featuring with many small dots), or you can add them programmatically from the viewer. We'll add them programmatically from the viewer for this example.

Reviewer: ```suggestion
You can add the layers using the "new points" button in the `layer list` panel of the viewer (left-most button featuring with many small dots), or you can add them programmatically from the viewer. We'll add them programmatically from the viewer for this example.
```

nbscreenshot(viewer)
```

> To add points you must enter add mode. This can be done by clicking on the `add` mode button in the top row of the control panel (2nd from the left, circle with a plus in it), or programmatically from the notebook.

Reviewer: ```suggestion
To add points you must enter add mode. This can be done by clicking on the `add` mode button in the top row of the `layer controls` panel (2nd from the left, circle with a plus in it), or programmatically from the notebook.
```

> Now start adding points, clicking once per cell, approximately in the center of the cell, with the appropriate `Points` layer selected. You can tell which `Points` layer is selected because it will be highlighted left in the layers list in the bottom left hand corner of the screen. You can rapidly switch between selected layers using the up and down keys. Also don't forget this is a z-slice so you should move up and down the slice, which can also be done with the left/ right key.

Reviewer: ```suggestion
Now start adding points, clicking once per cell, approximately in the center of the cell, with the appropriate `Points` layer selected. You can tell which `Points` layer is selected because it will be highlighted in the layers list in the bottom left hand corner of the screen. You can rapidly switch between selected layers using the up and down keys. Also don't forget this is a z-slice so you should move up and down the slice, which can also be done with the left/ right key.
```

You can also get the number of cells of each class and an array of their centers as follows:

```{code-cell} ipython3

> print('Number of diving cells: ', len(viewer.layers['dividing'].data))

Reviewer 2: Consider using f-strings.

`print(f"Number of diving cells: {len(viewer.layers['dividing'].data)}")`

> print('Number of diving cells: ', len(viewer.layers['dividing'].data))

Reviewer: ```suggestion
print('Number of dividing cells: ', len(viewer.layers['dividing'].data))
```

> print('Number of non-diving cells: ', len(viewer.layers['non-dividing'].data))

Reviewer: ```suggestion
print('Number of non-dividing cells: ', len(viewer.layers['non-dividing'].data))
```

nbscreenshot(viewer)
```

````{tip}
If you want to *remove* layers programmatically, you can do so by index, for

> example to remove the bottom most layer (0th index):

Reviewer: ```suggestion
example to remove the bottom-most layer (0th index):
```
